LICENSE

```diff
@@ -0,0 +1,150 @@
+   * [3.1.12.](#3.1.12)  Be an individual or entity, or a or a representative, agent, affiliate, successor, attorney, or assign of an individual or entity, that provides good or services to, or otherwise enters into any commercial contracts with, any local, state, or federal law enforcement agency;
+   * [3.1.13.](#3.1.13)  Be an individual or entity, or a representative, agent, affiliate, successor, attorney, or assign of an individual or entity which conducts military activities that discriminates against individuals on the basis of on the basis of race, ethnicity, caste, nationality, phenotype or genotype or perpetuates state-sanctioned violence;
+     * [3.1.13.1.](#3.1.11.2)  Be party to any commercial transactions with the any military that perpetuates state-sanctioned violence;
+   * [3.1.14.](#3.1.12)  Subject any person to arbitrary interference with a person's privacy, family, home, or correspondence without the express written consent by the person;
+     * [3.1.14.1.](#3.1.14.1)  Facilitate discrimination through removal from or prevent meaningful access to shelter based on arbitrary, inconsistent, or unclear requirements;
+     * [3.1.14.2.](#3.1.14.2)  Impose fines or fees, inadequate service, or otherwise punitive measures in housing based on discriminatory practices or policies;
+   * [3.1.15.](#3.1.15)  Arbitrarily deprive any person of his/her/their property;
+   * [3.1.16.](#3.1.16)  Forcibly remove indigenous peoples from their lands or territories or take any action with the aim or effect of dispossessing indigenous peoples from their lands, territories, or resources, including without limitation the intellectual property or traditional knowledge of indigenous peoples, without the free, prior, and informed consent of indigenous peoples concerned;
+   * [3.1.17.](#3.1.15) Perpetuate inequality in healthcare services for people on the basis of race, ethnicity, nationality, caste, phenotype or genotype, including, but not limited to:  
+     * [3.1.17.1.](#3.1.17.1)  Claiming medical devices, treatments, or procedures, which are untested or tested and known to be harmful,  as sufficient and exhaustive medical care without express written consent by the patient, or someone with authority to consent on behalf of the patient, acknowledging the unknown or harmful properties of the medical device, treatment, or procedure, and authorizing its use;
```

Comment:
"Claiming" needs to be "claim" in order to fit with the grammar of the section

Reply:
this section would be "including, but not limited to claiming medical devices...", "including, but not limited to using race-adjusted...", "including but not limited to, profiling..." 

I think this one works, but it is odd. I also see an extra space in there. I'll remove that.